spec/spec/spec_helper.rb

```diff
@@ -0,0 +1,89 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
...seems to still be in the pull request. Let me just clone down and get all the git stuff straight. I'll probably fork into a new PR.